torch/__init__.py

```diff
@@ -1274,6 +1274,15 @@ def _dtype(self):
 # The _tensor_classes set is initialized by the call to _C._initialize_tensor_type_bindings()
 _tensor_classes: Set[Type] = set()
 
+from enum import Enum
+
+class DefaultLoadEndian(Enum):
+    NATIVE = 1
+    LITTLE = 2
+    BIG = 3
+
+DEFAULT_LOAD_ENDIAN = DefaultLoadEndian.LITTLE
```

Comment:
Since this differs from default behavior we would have to go through a BC-breaking process here (see https://github.com/pytorch/pytorch/wiki/PyTorch's-Python-Frontend-Backward-and-Forward-Compatibility-Policy#minimizing-the-disruption-of-bc-breaking-changes)

I think it makes sense to set the default as NATIVE and users can change it via `torch.serialization.set_default_load_endianness` if necessary

Reply:
What would going through this process look like for this case?

And it would break compatibility only for big endian systems, because on little endian ones 'native' = 'little endian'.